src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -510,7 +510,7 @@ public function setBodyFromClosure(Closure $newBody) : void
      */
     public function setBodyFromString(string $newBody) : void
     {
-        $this->node->stmts = $this->loadStaticParser()->parse('<?php ' . $newBody);
+        $this->node->stmts = $this->loadStaticParser()->parse('<?php ' . $newBody) ?: [];
```

Comment:
Hmm... the parser should never yield non-arrays...

Reply:
In *theory* it can yield `null` just based on the return type.